.idea/.generators

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Hey. I sent new pull request. Fixed what you've commented. Please check.
And let me know if I can finish task.

Thanks,
Deimos,

On Wed, Jul 27, 2016 at 1:58 PM, Jason Denney <notifications@github.com>
wrote:

> In .idea/.generators
> <https://github.com/spacemunkay/BikeShed/pull/112#discussion_r72522227>:
>
> > @@ -0,0 +1,8 @@
> > +<?xml version="1.0" encoding="UTF-8"?>
>
> I would still like all .idea/ files to not be included in the source code.
> Each developer can create them if they desire.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/spacemunkay/BikeShed/pull/112/files/c9ee26ac0820b1b6749288780f296994e210f005#r72522227>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ATHQ9KyxQenhRrBQVZtNkKvDCMKX7Palks5qZ8ZkgaJpZM4JTvD->
> .
>